tagmanagementdispatcher/src/main/java/com/tealium/tagmanagementdispatcher/WebViewLoader.kt

```diff
@@ -410,6 +412,14 @@ class WebViewLoader(
         registerNewSessionIfNeeded(sessionId)
     }
 
+    override fun onInstanceShutdown(name: String, instance: WeakReference<Tealium>) {
+        try {
+            webView.destroy()
```

Comment:
this should only be called on the main thread - should already have access to the main thread in this class to wrap the call to the webview.